server/domains/list/entity.go

```diff
@@ -0,0 +1,129 @@
+package list
+
+import (
+	"database/sql"
+
+	"github.com/h3poteto/fascia/config"
+	"github.com/h3poteto/fascia/lib/modules/logging"
+	"github.com/h3poteto/fascia/server/domains/list_option"
+
+	"github.com/pkg/errors"
+)
+
+// List is a entity for list.
+type List struct {
+	ID             int64
+	ProjectID      int64
+	UserID         int64
+	Title          sql.NullString
+	Color          sql.NullString
+	ListOptionID   sql.NullInt64
+	IsHidden       bool
+	infrastructure Repository
+}
+
+type Repository interface {
```

Comment:
__[Golint]__ exported type Repository should have comment or be unexported

<sub>posted by Sider</sub>